js/main.js

```diff
@@ -54,23 +51,31 @@ function renderEntryDomTree(entry) {
 
   var liAnEntry = document.createElement('li');
   liAnEntry.setAttribute('class', 'row an-entry');
+  liAnEntry.setAttribute('data-entry-id', entry.entryId);
 
   liAnEntry.append(divImageSection, divInputSection);
 
   return liAnEntry;
 }
 window.addEventListener('DOMContentLoaded', function () {
-  var storageLocal = localStorage.getItem('code-journal-input');
-  if (storageLocal !== null) {
-    var storageData = JSON.parse(storageLocal);
-    data.view = storageData.view;
-    data.entries = storageData.entries;
-    data.editing = storageData.editing;
-    data.nextEntryId = storageData.nextEntryId;
-  }
-
   for (var ent = 0; ent < data.entries.length; ent++) {
     var list = renderEntryDomTree(data.entries[ent]);
     entryHolder.appendChild(list);
   }
 });
+
+var buttonNewEntry = document.getElementById('new-entry');
+buttonNewEntry.addEventListener('click', function () {
+  changeView('entry-form');
+});
+
+function changeView(view) {
+  if (view === 'entry-form') {
+    document.querySelector('[data-view=entry-form]').style.display = 'flex';
+    document.querySelector('[data-view=entries]').style.display = 'none';
```

Comment:
You should do these with class names, rather than adjusting the style directly. Remember, this is equivalent to inline styling, and inline styling is _the worst_